docs/poc2023q1/edge-scheduler.md

```diff
@@ -65,7 +61,7 @@ singleplacementslice.edge.kcp.io/test-1   5m54s
 ```
 
 Edit then delete the EdgePlacement.
-```
+```console
```

Comment:
Is there a place where the meaning of "console" here is defined?

Reply:
I think so.
https://github.com/github/linguist/blob/master/lib/linguist/languages.yml

Reply:
Thanks!